pkg/tests/integration_test.go

```diff
@@ -63,15 +63,6 @@ func TestInvalidResourceObject(t *testing.T) {
 	})
 }
 
-//nolint:paralleltest // uses parallel programtest
```

Comment:
Behaviour change: previous type mismatches will now be solved by prioritising `config` over `configuration`.

Reply:
I think this is fine.  `configuration` is deprecated, and having both is just a user error.  I *could* be confusing, but I think that's okay.